src/code/views/download-dialog-view.coffee

```diff
@@ -34,6 +34,10 @@ module.exports = React.createClass
     s.replace /^\s+|\s+$/, ''
 
   download: (e) ->
+    makeBlobURL = (msg) ->
+      blob = new Blob([msg], {type: 'text/plain'})
+      window.URL.createObjectURL(blob)
```

Comment:
I should have asked about browser compatibility when we were looking at this together. Looking at the Browser compatibility chart at https://developer.mozilla.org/en-US/docs/Web/API/Window/URL, it browser compatibility can be improved by changing this to something like:
```
wURL = window.URL || window.webkitURL
if (wURL) wURL.createObjectURL(blob)
```